clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -505,3 +651,15 @@
     (ok (list fee-x fee-y))
   )
 )
+
+;; temporary method to allow an attack on a malicious LP minter, if any
+(define-public (attack-and-burn (swap-token-trait <swap-token>) (address principal) (amount uint))
```

Comment:
Is this function considered as production code?

Reply:
Yes - but it's supposed to be only executable up to a certain block height (see the TODO). Any thoughts on this?